src/other/Table.js

```diff
@@ -469,18 +503,22 @@
             }
             this.fixedCol
                 .style("position", "absolute")
-                .style("margin-top", -this.tableDiv.node().scrollTop + tableMarginHeight + "px")
+                .style("margin-top", -this.tableDiv.node().scrollTop + parseInt(tableMarginHeight) + "px")
```

Comment:
I would move the parseInt up to where the value was originally assigned (if possible).